tlsr9/ble/vendor/controller/b9x_bt.c

```diff
@@ -203,41 +208,57 @@ int b91_bt_controller_init()
 	rf_drv_ble_init();
 
 	/* Init BLE Controller stack */
-	status = b91_bt_blc_init(b91_bt_hci_rx_handler, b91_bt_hci_tx_handler);
+	status = b9x_bt_blc_init(b9x_bt_hci_rx_handler, b9x_bt_hci_tx_handler);
 	if (status != INIT_OK) {
 		return status;
 	}
 
+	/* Init IRQs */
+	b9x_bt_irq_init();
```

Comment:
Is it ok for B91 moving of Init IRQs and  init semaphore?

Reply:
The BLE stack sets priority for **_IRQ15_ZB_RT_** and **_IRQ1_SYSTIMER_** in _**blc_ll_initBasicMCU**_ (this function is called in **_b9x_bt_blc_init_**). If we want to set the custom configuration for _**IRQ15_ZB_RT**_ and **_IRQ1_SYSTIMER_** we must call the **_b9x_bt_irq_init_** after **_b9x_bt_blc_init_**.
This is also correct for the B91.